tests/templates/kuttl/smoke/32-assert.yaml

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
I would rename this to 31-assert.yaml. Better to resolve a conflict than testing the same thing twice.